### PR TITLE
feat(uwucommand): added uwu command

### PR DESCRIPTION
**Pull request by Jessuhh**
Added the uwu command to UwUify a provided sentence

image

Shamelessly copied from https://github.com/Joehoel/compagnon/blob/master/src/commands/fun/uwu.ts

### CODE REVIEW

File: src/commands/UwuCommand.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    run: async (interaction) => {
+        try {
+            interaction.reply(
+                uwu.uwuifySentence(interaction.options.getString('sentence')!),
+            );
+        } catch (err) {
+            console.error(err);
+            interaction.reply({
+                content: 'Failed UwUifying your text OwO',
```
Comment (rickklaasboer) [inline src/commands/UwuCommand.ts:27]: thanks i hate it

**Review by NiekKleverkamp — CHANGES_REQUESTED** [review]
Gewoon verwijderen

**Review by rickklaasboer — APPROVED** [review]
LGTBQ once status checks pass

File: src/commands/UwuCommand.ts (hunk 1 above)
Comment (Jessuhh) [reply src/commands/UwuCommand.ts:27]: You're welcome! :D

**Comment by Jessuhh** [comment]
>

**Pull request merged**